apache-rat-core/src/main/java/org/apache/rat/ReportConfiguration.java

```diff
@@ -114,16 +174,18 @@ public void setReportable(IReportable reportable) {
     }
 
     /**
-     * @return the Supplier of the InputStream that is the XSLT style sheet to style the report with.
+     * @return the Supplier of the InputStream that is the XSLT style sheet to style
+     * the report with.
      */
     public IOSupplier<InputStream> getStyleSheet() {
         return styleSheet;
     }
 
     /**
-     * Sets the style sheet for custom processing.
-     * The IOSupplier may be called multiple times, so the input stream must be 
-     * able to be opened and closed multiple times.
+     * Sets the style sheet for custom processing. The IOSupplier may be called
+     * multiple times, so the input stream must be able to be opened and closed
+     * multiple times.
```

Comment:
btw: I guess that this new behaviour is the cause of https://issues.apache.org/jira/browse/RAT-344 ;)